README.md

```diff
@@ -252,11 +252,51 @@ For the full test suite, including long-running tests, use:
 cargo test --release
 ```
 
-Additional functional tests are available. Install dependencies and run the test script with:
+#### Functional tests
+
+Additional functional tests are available. 
+
+* Install [poetry dependencie manager](https://python-poetry.org/docs/#installation). There're 
```

Comment:
`s/dependencie/dependencies/`

Also, this contraption is weird when starting a sentence. Use the explicit `There are`

Reply:
Fixed